src/gui/editors/PianoRoll.cpp

```diff
@@ -761,7 +742,10 @@ void PianoRoll::fitNoteLengths(bool fill)
 		}
 		if (ref == refNotes.end())
 		{
-			if (!fill) { break; }
+			if (!fill)
+			{
+				break;
+			}
```

Comment:
I say go with this because I think branches always need indentation.
Plus it would be consistent with long ifs.

Reply:
Keep the exception, because the current style uses an absurd amount of newlines.

Reply:
As I understand, you (Veratil, Spekular) want the brackets in one line? I set `AllowShortBlocksOnASingleLine: Always` in `.clang-format`, but it had no effects. I'm not sure if this is possible with `ColumnLimit: 0` (or it may be a tool bug that this special `if` is not considered)

Reply:
I'm fine with combining the brackets inline in all places because it doesn't waste vertical space. We can easily tell by indentation where a block is, so an entire line dedicated to a single bracket within a 3000 line file can save quite a bit of vertical space.

As for the setting not working, it's probably because of some other combination of settings. I've noticed the tool doesn't work sometimes on a second (or third, etc) pass after changing some settings, too. That might be where things are failing to work.

Reply:
But I solved it using `AllowShortIfStatementsOnASingleLine`: `if (!fill) { break; }`. I think this is what you wanted?

Reply:
Ah, yeah! That's good. 😄